web_src/css/modules/label.css

```diff
@@ -0,0 +1,571 @@
+}
+.ui.basic.labels a.purple.label:hover,
+a.ui.ui.ui.basic.purple.label:hover {
+  background: transparent;
+  border-color: var(--color-purple-dark-1);
+  color: var(--color-purple-dark-1);
+}
+.ui.pink.labels .label,
+.ui.ui.ui.pink.label {
+  background: var(--color-pink);
+  border-color: var(--color-pink);
+  color: var(--color-white);
+}
+
+.ui.pink.labels a.label:hover,
+a.ui.ui.ui.pink.label:hover {
+  background: var(--color-pink-dark-1);
+  border-color: var(--color-pink-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .pink.label,
+.ui.ui.ui.basic.pink.label {
+  background: transparent;
+  border-color: var(--color-pink);
+  color: var(--color-pink);
+}
+.ui.basic.labels a.pink.label:hover,
+a.ui.ui.ui.basic.pink.label:hover {
+  background: transparent;
+  border-color: var(--color-pink-dark-1);
+  color: var(--color-pink-dark-1);
+}
+.ui.brown.labels .label,
+.ui.ui.ui.brown.label {
+  background: var(--color-brown);
+  border-color: var(--color-brown);
+  color: var(--color-white);
+}
+
+.ui.brown.labels a.label:hover,
+a.ui.ui.ui.brown.label:hover {
+  background: var(--color-brown-dark-1);
+  border-color: var(--color-brown-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .brown.label,
+.ui.ui.ui.basic.brown.label {
+  background: transparent;
+  border-color: var(--color-brown);
+  color: var(--color-brown);
+}
+.ui.basic.labels a.brown.label:hover,
+a.ui.ui.ui.basic.brown.label:hover {
+  background: transparent;
+  border-color: var(--color-brown-dark-1);
+  color: var(--color-brown-dark-1);
+}
+.ui.grey.labels .label,
+.ui.ui.ui.grey.label {
+  background: var(--color-label-bg);
+  border-color: var(--color-label-bg);
+  color: var(--color-label-text);
+}
+
+.ui.grey.labels a.label:hover,
+a.ui.ui.ui.grey.label:hover {
+  background: var(--color-label-hover-bg);
+  border-color: var(--color-label-hover-bg);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .grey.label,
+.ui.ui.ui.basic.grey.label {
+  background: transparent;
+  border-color: var(--color-label-bg);
+  color: var(--color-label-text);
+}
+.ui.basic.labels a.grey.label:hover,
+a.ui.ui.ui.basic.grey.label:hover {
+  background: transparent;
+  border-color: var(--color-label-hover-bg);
+  color: var(--color-label-hover-bg);
+}
+.ui.black.labels .label,
+.ui.ui.ui.black.label {
+  background: var(--color-black);
+  border-color: var(--color-black);
+  color: var(--color-white);
+}
+
+.ui.black.labels a.label:hover,
+a.ui.ui.ui.black.label:hover {
+  background: var(--color-black-dark-1);
+  border-color: var(--color-black-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .black.label,
+.ui.ui.ui.basic.black.label {
+  background: transparent;
+  border-color: var(--color-black);
+  color: var(--color-black);
+}
+.ui.basic.labels a.black.label:hover,
+a.ui.ui.ui.basic.black.label:hover {
+  background: transparent;
+  border-color: var(--color-black-dark-1);
+  color: var(--color-black-dark-1);
+}
```

Comment:
Same
```suggestion
```

Additionally, `web_src/css/base.css` has unused rules:
```css
1638:.ui.basic.labels .black.label,
1639:.ui.ui.ui.basic.black.label {
```

Reply:
Those selectors are not there in base.css.